cypress/Shared/Utilities.ts

```diff
@@ -286,8 +260,8 @@ export class Utilities {
         cy.get(element, { timeout: timeout }).should('not.have.attr', 'readonly', 'readonly')
     }
 
-    public static validateTCPopValueCheckBoxes(measureScoreValue: string | string[]): void {
-        switch ((measureScoreValue.valueOf()).toString()) {
+    public static validateTCPopValueCheckBoxes(measureScoreValue: string): void {
```

Comment:
I couldn't see a use-case for an array here. These changes simplify the process significantly.